spring-cloud-dataflow-docs/src/main/asciidoc/streams.adoc

```diff
@@ -82,6 +82,68 @@ The xref:getting-started#getting-started[Getting Started] section shows you how
 
 Note that the shell calls the Data Flow Servers' REST API. For more information on making HTTP requests directly to the server, consult the <<api-guide, REST API Guide>>.
 
+=== Stream Application DSL
+
+The Stream Pipeline DSL described in the previous section automatically sets the input and output binding properties of each Spring Cloud Stream application.
+This can be done because there is only one input and/or output destination in a Spring Cloud Stream application that uses the provided binding interface of a `Source`, `Processor`, or `Sink`.
```

Comment:
Try to avoid "and/or" please. This one can be "or". You can remove "provided", too. Given that there's no opposite (an unprovided interface?), the word makes no actual distinction and is unnecessary.